### ISSUE TITLE
Command prompt tab completion

### ISSUE THREAD

**Issue by dan1982code** [nvaccess/nvda#6095]
In a command prompt, Create a directory with two subdirectories called w14 and w15. Keep pressing tab; the command prompt tab-completes between the two subdirectories (w14 and w15) but NVDA does not read these changes except for the first. If the tab-completed names are very different, then NVDA does read them.

**Comment by Adriani90** [comment]
@dan1982code are you still having this issue with last NVDA Alpha?

**Comment by dan1982code** [comment]
Yes. It's as if when only the final character changes, NVDA misses it and speaks nothing. Thanks!

**Comment by Adriani90** [comment]
cc: @leonardder

**Comment by Adriani90** [comment]
cc: @codeofdusk is this maybe also fixed in UIA environment?

**Comment by codeofdusk** [comment]
> cc: @codeofdusk is this maybe also fixed in UIA environment?

No, since we use the same diffing algorithm (in fact, nothing is announced in UIA since we clear the words buffer for tab, which is dropping some things). I may revert that to improve consistency with other consoles (i.e. NVDA's python console).

**Comment by codeofdusk** [comment]
This is essentially a duplicate of #3200.

**Comment by seanbudd** [comment]
as #3200 is fixed, is this still an issue?
